percona-server.80/README.md

```diff
@@ -2,15 +2,20 @@
 
 # What is Percona Server?
 
-Percona Server is an enhanced drop-in replacement for MySQL. With Percona Server, Your queries will run faster and more consistently. You will consolidate servers on powerful hardware. You will delay sharding, or avoid it entirely.
+Percona Server is an enhanced drop-in replacement for MySQL. With Percona 
+Server, your queries will run faster and more consistently. You will consolidate 
```

Comment:
why we need a split paragraph to multiple lines?

Reply:
Good morning. Fixed. Thanks.